app/stac_api/admin.py

```diff
@@ -27,6 +28,12 @@ class LandingPageAdmin(SingletonModelAdmin):
     inlines = [LandingPageLinkInline]
 
 
+@admin.register(ConformancePage)
+class ConformancePageAdmin(SingletonModelAdmin):
+    model = ConformancePage
+    extra = 0
```

Comment:
I think these are not needed ? Could be simply
```
@admin.register(ConformancePage)
class ConformancePageAdmin(SingletonModelAdmin):
    pass
```
The model is redundant because already in register decorator, but this needs to be quickly tested.